admin/scripts/modules/penize/penize.xtpl

```diff
@@ -1,10 +1,128 @@
 <!-- begin:penize -->
 
-<h1>Peníze</h1>
+<h1>Penize</h1>
```

Comment:
Tohle je asi pokus o změnu názvu v menu?

Vrátil bych diakritiku, pokud to nemá skrytý význam.

Reply:
Překlep. Opraveno.